benchmark/single-source/IndexPathTest.swift

```diff
@@ -0,0 +1,163 @@
+//===--- IndexPathTest.swift ----------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import TestsUtils
+import Foundation
+
+let size = 200
+let tags: [BenchmarkCategory] = [.validation, .api, .IndexPath]
+
+public let IndexPathTest = [
+  BenchmarkInfo(name: "IndexPathSubscriptMutation", 
+  runFunction: { n in run_IndexPathSubscriptMutation(n * 1000, size)}, 
+  tags: tags),
+  BenchmarkInfo(name: "IndexPathSubscriptRangeMutation", 
+  runFunction: { n in run_IndexPathSubscriptRangeMutation(n * 1000, size)}, 
+  tags: tags),
+
+  BenchmarkInfo(name: "IndexPathMaxBeginning", 
+  runFunction: { n in run_IndexPathMaxBeginning(n * 1000)}, 
+  tags: tags),
+  BenchmarkInfo(name: "IndexPathMaxMiddle", 
+  runFunction: { n in run_IndexPathMaxMiddle(n * 1000)}, 
+  tags: tags),
+  BenchmarkInfo(name: "IndexPathMaxEnd", 
+  runFunction: { n in run_IndexPathMaxEnd(n * 1000)}, 
+  tags: tags),
+
+  BenchmarkInfo(name: "IndexPathMinBeginning", 
+  runFunction: { n in run_IndexPathMinBeginning(n * 1000)}, 
+  tags: tags),
+  BenchmarkInfo(name: "IndexPathMinMiddle", 
+  runFunction: { n in run_IndexPathMinMiddle(n * 1000)}, 
+  tags: tags),
+  BenchmarkInfo(name: "IndexPathMinEnd", 
+  runFunction: { n in run_IndexPathMinEnd(n * 1000)}, 
+  tags: tags),
+]
+
+@inline(__always)
+func indexPath(_ size: Int, reversed: Bool = false) -> IndexPath {
+    let indexes = Array(0..<size)
+    return IndexPath(indexes: reversed ? indexes.reversed() : indexes)
+}
+
+@inline(__always)
+func indexPath(_ size: Int, middle: Int) -> IndexPath {
+    var indexes = Array(0..<size)
+    indexes.insert(middle, at: (indexes.count - 1) / 2)
+    return IndexPath(indexes: indexes)
+}
+
+// Subscript Mutations
+
+@inline(__always)
+func subscriptMutation(
+  n: Int,
+  mutations: Int,
+  mutate: (inout IndexPath, Int) -> ()) {
+    for _ in 0..<n {
+      var ip = indexPath(size)
+
+      for i in 0..<mutations {
+        mutate(&ip, i)
+      }
+
+      blackHole(ip)
+    }
```

Comment:
```suggestion
  mutate: (inout IndexPath, Int) -> ()
) {
  for _ in 0..<n {
    var ip = indexPath(size)
    for i in 0..<mutations {
      mutate(&ip, i)
    }
    blackHole(ip)
  }
```

Part of what I meant was that the standard library style here is to put the closing parenthesis for the parameter list on its own line. This helps keep the indenting in the body to be two spaces without aligning weirdly with what comes before.

Reply:
I see! Thanks for that. I went ahead and ran swift-format with the config you suggested, and applied your suggestions. It formatted other parts of the code, but I assume it follows the style.